dashboard/app/views/devise/registrations/_finish_sign_up.html.haml

```diff
@@ -11,20 +11,27 @@
   - @page_title = t('activerecord.attributes.user.finish_sign_up_header_error')
 - else
   - @page_title = t('activerecord.attributes.user.finish_sign_up_header')
+- is_lti = Policies::Lti.lti?(@user)
+- show_lti_warning = is_lti && @user.teacher?
+- issuer_url = is_lti ? Policies::Lti.issuer(@user) : nil
+- issuer_name = issuer_url ? Policies::Lti.issuer_name(issuer_url) : t('lti.title')
```

Comment:
I like how you extracted the logic up here so it can be reused instead of having to call `Policies::Lti.lti?(@user)` multiple times.